### PR TITLE
Update readme and description

### PR DESCRIPTION
**Pull request by berkcebi**
✏️

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,39 +1,38 @@
...
 
 ## Installation
 
-[Download](https://marketplace.visualstudio.com/items?itemName=zeplin.zeplin) Zeplin from Visual
-Studio Code extension marketplace
+1. [Install](https://marketplace.visualstudio.com/items?itemName=zeplin.zeplin) Zeplin from Visual Studio Code extension marketplace.
+2. Open the new Zeplin sidebar.
+3. Login to Zeplin.
 
 ## Usage
```
Comment (berkcebi) [inline README.md:11]: I'll add screenshots here once marketing materials are ready.

File: README.md [hunk 2]
```diff
@@ -1,39 +1,38 @@
...
-All these steps can either be:
+### Manage Connected Components configuration
+
+[Connected Components](https://zpl.io/connected-components) in Zeplin lets you access components in your codebase directly on designs in Zeplin, with links to Storybook, GitHub and any other source of documentation based on your workflow. 🧩
 
-- Run as a command, default shortcut is “Command/Ctrl + Shift + P”.
-- Run by clicking Code Lens/Hover links in the configuration file.
+Zeplin for Visual Studio Code makes it easier to create the JSON configuration file that connects components in your codebase to their design counterparts in Zeplin. Check out our getting [started guide for Connected Components](https://github.com/zeplin/connected-components-docs/).
 
 ## Contributing and Issue Tracking
```
Comment (berkcebi) [inline README.md:38]: Should we remove the Roadmap section below and create issues for them if necessary?

File: README.md [hunk 3]
```diff
@@ -1,39 +1,38 @@
...
+Get started by adding a project. Once you add one, all the screens and components in the project will be listed in the sidebar. You can quickly open each screen or component in Zeplin by clicking the “Open in Zeplin” icon in each row. Similar to adding projects, you can also add styleguides and access all their components.
 
-Zeplin's Visual Studio Code extension makes it easier to create the JSON configuration file that connects components in your codebase to their design counterparts in Zeplin.
+You can also pin screens or components you frequently visit, and they will be listed under the “Pinned” panel.
 
-☝️ *Note that using Visual Studio Code and the extension is optional and the JSON configuration file can always be created manually in any text editor.*
+### Follow designs changes
 
-![Preview](resources/docs/sample.gif)
+The Zeplin sidebar lets you to see which designs have been updated since you last visited. Once you add a project, Zeplin will list changes to screens and components under the “Activities” panel.
```
Comment (fonurr) [inline README.md:23]: Is should be "Activitiy" instead of "Activities"

File: README.md [hunk 4]
```diff
@@ -1,39 +1,38 @@
-# Zeplin Visual Studio Code Extension
+# Zeplin for Visual Studio Code
 
-Visual Studio Code extension for Zeplin 🔌
+Zeplin for Visual Studio Code lets you quickly access the designs you’re working on, follow design changes, open the Jira issues they’re attached to—all from within VS Code. 🔌
```
Comment (fonurr) [inline README.md:3]: We may add a space before "-" if I am not wrong

**Comment by fonurr** [comment]
rebased and fixed comments

**Pull request merged**